modules/appmenu-gtk3-module.nix

```
@@ -13,8 +13,12 @@ in
       };
   };
   config = {
-    environment.profileRelativeSessionVariables = lib.mkIf cfg.enable {
+    environment.sessionVariables = lib.mkIf cfg.enable {
       XDG_DATA_DIRS = [ "${pkgs.appmenu-gtk3-module}/share/gsettings-schemas/appmenu-gtk3-module-0.7.6" ];
     };
+
+    environment.systemPackages = with pkgs; [ appmenu-gtk3-module ];
+
+    environment.variables."UBUNTU_MENUPROXY" = "1";
```

```suggestion
    environment.variables.UBUNTU_MENUPROXY = "1";
```

For attrset's keys, `.x` is the same as `."x"`. Recently, I've learned that `."${y}" is the same as `.${y}`.